app/scripts/related-entity-transform.js

```diff
@@ -120,23 +151,59 @@ var relatedEntityTransform = (function() {
                     "url": "http://someurlhere.com",
                     "body": "description text here",
                     "addresses": ["Los Angeles"],
+                    """if phone exists"""
+                    "phone": "<phonenumber>",
+                    """if email exists"""
+                    "email": "<email>"
                     "date": "2012-04-23T18:25:43.511Z"
                 }
             }
         */
         var webpageObj = {
             _id: record._id,
             _type: record._type,
-            title: _.get(record, '_source.publisher.name', 'Publisher N/A'),
-            subtitle: _.get(record, '_source.name', 'Title N/A'),
+            title: _.get(record, '_source.name', 'Title N/A'),
+            subtitle: [_.get(record, '_source.publisher.name', 'Publisher N/A')],
+            offer: _.get(record, '_source.mainEntity.uri'),
             details: {
                 url: _.get(record, '_source.url'),
                 body: _.get(record, '_source.description'),
                 addresses: getAddressArray(record),
-                date: _.get(record, '_source.dateCreated')
+                _sortedKeys: ['url', 'body', 'addresses', 'phone', 'email', 'date']
             }
 
         };
+        var mentions = _.get(record, '_source.mentions');
+        if(mentions) {
+            phoneEmails = commonTransforms.getEmailAndPhoneFromMentions(mentions);
+            if(phoneEmails.phones.length > 0) {
+                var sep = "";
+                phones = "";
+                phoneEmails.phones.forEach(function(phoneObj) {
+                    phones += sep + phoneObj.title;
+                    sep = ", ";
```

Comment:
You don't have to do this but in case you're interested this is a neat trick that would remove the need to create/use the "sep" variable:
```
phoneEmails.emails.forEach(function(emailObj, index) {
  emails += (index > 0 ? ", " : "") + emailObj.title;
});
````